idunn/blocks/reviews.py

```diff
@@ -37,13 +37,13 @@ def build_rating_bubble_star_url(rating):
     return base_url
 
 
-def build_review(review: dict, source_url: str) -> Review:
+def build_review(review: dict, source_base_url: str) -> Review:
     return Review(
         date=review["DatePublished"],
         rating=review["Rating"],
         rating_bubble_star_url=build_rating_bubble_star_url(review["Rating"]),
-        url="".join([source_url, review["ReviewURL"]]),
-        more_reviews_url="".join([source_url, review["MoreReviewsURL"]]),
+        url="".join([source_base_url, review["ReviewURL"]]),
+        more_reviews_url="".join([source_base_url, f"{review['MoreReviewsURL']}{'#REVIEWS'}"]),
```

Comment:
```suggestion
        more_reviews_url="".join([source_base_url, f"{review['MoreReviewsURL']}#REVIEWS"]),
```

Feels like I did the PR, I'm usually the one ending up with weird deviations :stuck_out_tongue:

Reply:
:sweat_smile: